chispa/__init__.py

```diff
@@ -46,28 +49,26 @@
 
 
 class Chispa:
-    def __init__(self, formats: FormattingConfig | None = None, default_output=None):
+    def __init__(self, formats: FormattingConfig | None = None) -> None:
```

Comment:
Is it a breaking change? Why did we drop `default_output`?

Reply:
The flag does not seem to be used anywhere.